### PR TITLE
first attempt at tip calculator

### PR DESCRIPTION
**Pull request by esaari**

### CODE REVIEW

File: Tip_Calculator/script.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+//Validate user input
+var validate = function (input) {
+    if (input === null || input === false || input === "") {
+        document.write("You need to enter a valid number!");
+        throw("ERROR: Invalid user input");
+    }
+};
+
+//Collect our user inputs
+var mealAmount = parseFloat(prompt("Enter the cost of your meal :"));
```
Comment (esaari) [inline Tip_Calculator/script.js]: I attempted to wrap this in the validate function above, but I was still getting `NaN` coming through.  What I attempted: 
`var mealAmount = validate(parseFloat(prompt("Enter the cost of your meal :")));`

File: Tip_Calculator/script.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+//Validate user input
+var validate = function (input) {
+    if (!input) {
+        document.write("You need to enter a valid number!");
+        throw("ERROR: Invalid user input");
+    }
+    else {
+        return input;
```
Comment (esaari) [inline Tip_Calculator/script.js:8]: Got it, realized I wasn't returning anything on success.

**Comment by Airswoop1** [comment]
@esaari  Looks good to me. Can you add me as a collaborator to the repo so I can merge pull requests? Thanks

**Comment by esaari** [comment]
Done.  I opened this PR last week, but figured you would've gotten an email because you were a watcher, didn't know I needed to add you as a collaborator.

**Pull request merged**